Supply multiple beacon node endpoints. (#386)
ConsenSys/doc.teku: docs/Reference/CLI/Subcommands/Validator-Client.md

Run a validator client that connects to a remote beacon node.

## `beacon-node-api-endpoint`, `beacon-node-api-endpoints`

=== "Syntax"

    ```bash
    teku vc --beacon-node-api-endpoint=<ENDPOINT>[,<ENDPOINT>...]...
    ```

=== "Example"

    ```bash
    teku vc --beacon-node-api-endpoint=http://192.138.10.12:5051,http://192.140.11.44:5051
    ```

=== "Environment variable"

    ```bash

>     TEKU_BEACON_NODE_ENDPOINT=http://192.138.10.12,http://192.140.11.44:5051

Reviewer: Should it not be TEKU_BEACON_NODE_API_ENDPOINT?

    ```

=== "Configuration file"

    ```bash
    beacon-node-api-endpoint: ["http://192.138.10.12","http://192.140.11.44:5051"]
    ```

Endpoint of the beacon node's REST API. You can configure multiple beacon nodes by providing a
comma-separated list of beacon node API endpoints.

> If multiple beacon node endpoints are configured, the first one will be used as primary and others

Reviewer 2: ```suggestion
If multiple beacon node endpoints are configured, the first one is used as primary and others
```
